### PR TITLE
add PKGBUILD and workflow for DEB package

### PR DESCRIPTION
**Pull request by zocker-160**
PR adds a PKGBUILD file which can be used to generate a DEB for Debian / Ubuntu using makedeb or a pkg file for Arch using makepkg.

I have also updated the workflow to generate the deb file, it does not get added to the release upload, since I do not know what your naming convention for those files is.

Makedeb will generate a file called `hpk_0.3.9-1_amd64.deb` and can be easily installed using `apt install`.

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+build() {
+    cd $srcdir/$pkgname
+    
+    cargo build --release
+}
+
+package() {
+    cd $srcdir/$pkgname
+
+    install -D -m755 target/release/hpk -t "$pkgdir"/usr/bin
```
Comment (nickelc) [inline PKGBUILD:25]: can you also include the license file?

Comment (zocker-160) [reply PKGBUILD:25]: of course

Comment (zocker-160) [reply PKGBUILD:25]: added

**Comment by nickelc** [comment]
It's good as it is.
I'll finish the release workflow before I finally tag the next version.

**Pull request merged**